packages/grafana-ui/src/components/Forms/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -11,6 +12,23 @@ const getRadioButtonGroupStyles = () => {
       flex-wrap: nowrap;
       position: relative;
     `,
+    'radio-group': css`
```

Comment:
Can we use camelCase for the style names?

Reply:
Ah, sure, will do